pkg/containerutil/cp_linux.go

```diff
@@ -54,13 +55,27 @@ func CopyFiles(ctx context.Context, client *containerd.Client, container contain
 		if !errdefs.IsNotFound(err) || rootlessutil.IsRootless() {
 			return err
 		}
-		root, cleanup, err = mountSnapshotForContainer(ctx, client, container, snapshotter)
-		if cleanup != nil {
-			defer cleanup()
+		if !container2host {
+			path = dst
+		} else {
+			path = src
 		}
+		// Check if path is in a volume
+		root, destination, err = getContainerMountInfo(ctx, container, path)
```

Comment:
this check concerns only `host  to container` copy , please put in the above if clause